src/main-process/main.js

```diff
@@ -39,9 +39,8 @@ if (args.resourcePath) {
 } else {
   const stableResourcePath = path.dirname(path.dirname(__dirname));
   const defaultRepositoryPath = path.join(
-    app.getPath('home'),
-    'github',
-    'atom'
+    // Setting the path for the app
+    app.getPath('home','pulsar-edit','pulsar')
```

Comment:
```suggestion
    app.getPath('home'),
    'pulsar-edit',
    'pulsar'
```
https://www.electronjs.org/docs/latest/api/app#appgetpathname

app.getPath takes a single string. path.join concatenates the array of strings together into a properly formatted path.

Though I would actually suggest using this
```suggestion
    app.getPath('home'),
    'github',
    'pulsar'
```
because the default git folder _is_ `github` when installing git (at least on windows). And I would personally discourage users from cloning repos directly into their home folder

Reply:
Ok will fix